libcxx/test/std/containers/container.adaptors/flat.set/allocator_mismatch.compile.fail.cpp

```diff
@@ -0,0 +1,21 @@
+//===----------------------------------------------------------------------===//
```

Comment:
This test (and the corresponding `std::set` test) should be a `.verify.cpp` test.

Reply:
Updated all the existing containers' `allocator_mismatch` tests in #8; PTAL!
After doing that, I found that this new test suffered from the exact failure mode of `.fail.cpp` tests — it wasn't testing what I thought it was! The third argument to `set` is an allocator, but the third argument to `flat_set` is supposed to be a container; so of course this test failed to compile as written. We could add a new test for `flat_set<int, less<int>, vector<int, allocator<long>>>`; but that's already covered by `vector/allocator_mismatch.verify.cpp`, so actually I think there's no longer anything to test here for `flat_set`.